<?xml version="1.0"?>

<launch>
  <!-- Start pattern generator node -->
  <node name="pattern_generator" pkg="l3_pattern_generator" type="pattern_generator_node" output="screen" />
</launch>
